example/android/app/src/main/java/com/example/visioncameraimagelabeler/MainApplication.java

```diff
@@ -18,13 +23,23 @@ public boolean getUseDeveloperSupport() {
 
         @Override
         protected List<ReactPackage> getPackages() {
-          return new ArrayList<ReactPackage>();
+          @SuppressWarnings("UnnecessaryLocalVariable")
+          List<ReactPackage> packages = new PackageList(this).getPackages();
+          // Packages that cannot be autolinked yet can be added manually here, for VisionCameraImageLabelerExample:
```

Comment:
can you help me understand why this lib cannot be auto linked yet?

Reply:
The comment doesn't mean much here, it's just boilerplate left from the React Native template. In this case the `vision-camera-image-labeler` library is manually linked to the internal example app there in order to enable testing changes live during development (see the [library template](https://github.com/callstack/react-native-builder-bob/blob/0bfd51ab2d550cf74ab3851021eee640b2d90b94/packages/create-react-native-library/templates/example/example/android/app/src/main/java/com/example/%7B%25-%20project.package%20%25%7D/MainApplication.java#L27) for reference)

The published `vision-camera-image-labeler` package can be auto linked as normal.